packages/ra-ui-materialui/src/form/SimpleForm.js

```diff
@@ -27,11 +27,26 @@ const SimpleForm = ({ initialValues, ...props }) => {
         props.save(values, finalRedirect);
     };
 
-    const finalInitialValues = {
+    let finalInitialValues = {
```

Comment:
It would probably be a good idea to extract all the *get initial values* code in a function to use in both `SimpleForm` and `TabbedForm`

Reply:
ok